@xen-orchestra/lite/src/components/form/FormByteSize.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <FormInputGroup>
+    <FormInput v-model="size" type="number" />
+    <FormSelect v-model="prefix">
+      <option
+        v-for="currentPrefix in availablePrefixes"
+        :key="currentPrefix"
+        :value="currentPrefix"
+      >
+        {{ currentPrefix }}B
```

Comment:
What about  `B/s` or `b` for example?

Reply:
Or when we'll need to, we could even pass a prop to completely overwrite the unit. But I agree, let's not try to solve problems we don't have right now, most of the size inputs will be in "B" anyway.

However, you could pass that "B" unit to `humanFormat` directly. Or was there a reason not to?

Reply:
I'm not sure to understand what you mean by "you could pass that "B" unit to `humanFormat` directly"

Reply:
I think I meant that `humanFormat` optionally takes a `unit` option and appends it to the returned value. But here, you only want "prefix + unit" and I don't think you can get that back from `humanFormat`'s raw output, so forget about it, it's fine like this.